schema/profiles/var-study-stmt-source.yaml

```diff
@@ -35,8 +36,8 @@ $defs:
       predicate:
         extends: predicate
         enum:
-        - positive
-        - negative
+        - supports_positive_diagnosis_for
```

Comment:
I'm going to defer to @ahwagner on these predicate term changes.  I do like the idea of making them consistent as you mentioned in your description of this PR. I do think we should stick to using camelCase instead of snake_case since we've started with camelCase in the other profiles (unless you've changed those as well). 
I'll defer approving this until @ahwagner weighs in since these profiles are in his territory.

Reply:
@ahwagner can you weigh in on this comment so we can modify and approve or simply approve as-is?

Reply:
agree with case conventions.

Need to give a little thought to the predicate conventions here. "Positive" means that the presence of the variant is a positive indicator for a disease diagnosis (inclusion criteria for a disease); "negative" means that the presence of the variant is negative indicator for a disease diagnosis (exclusion criteria for). Not sure I like the use of "negative diagnosis" and "positive diagnosis", though; that seems more like a test result than inclusion / exclusion criteria.

Reply:
Happy to come up with a different phrase for the predicate.  As long as it is a verb phrase that expresses the relationship between the variant and the disease in this statement types, and not just the word "positive".  

@ahwagner based on your clarifications would one of the the following work better:

- `positiveIndicatorFor`
- `positiveIndicatorForDisease`
- `positiveDiagnosticIndicatorFor`
- `positiveIndicatorForDiagnosisOf`

. . . or perhaps you have better suggestions?

Reply:
I will go with whatever verb phrase you think is best here Alex.  I'll update the PR to use your suggestion above for now.  We can always update the predicate labels later.